### DISCUSSION TITLE
Create database

### DISCUSSION THREAD

**Discussion by engahlw** [manuvarkey/GElectrical#14] [Q&A] [answered]
Could you provide the structure for creating the database? Sometimes we need to include equipment, these with different electrical and mechanical parameters, mainly due to use in different countries.

**Comment by manuvarkey** [comment, marked as answer]
Database files are `.csv` files with `;` as field delimiter. The first row gives the reference codes of the data fields corresponding to columns. Subsequent rows gives the data field values.

For overriding a user database, the corresponding `.csv` file from `database` folder of GElectrical installation (or from [https://github.com/manuvarkey/GElectrical/tree/master/gelectrical/database](https://github.com/manuvarkey/GElectrical/tree/master/gelectrical/database)) can be copied to the user library path (folder can be accessed from `Program settings` -> `Paths`) after making suitable modifications.
